### PR TITLE
[5][plugin] emailcloack Deprecated: mb_strpos()

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #42846 .

### Summary of Changes
don't cloack  if null

### Testing Instructions
see #42846 

### Actual result BEFORE applying this Pull Request

You'll get the error: Deprecated: mb_strpos()

### Expected result AFTER applying this Pull Request
no more Deprecated

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on 6f2c9b937c95124b9269446c283f46bc587eb719<br /><br />Patch works!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42865">issues.joomla.org/tracker/joomla-cms/42865</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 6f2c9b937c95124b9269446c283f46bc587eb719<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42865">issues.joomla.org/tracker/joomla-cms/42865</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42865">issues.joomla.org/tracker/joomla-cms/42865</a>.</sub>

File: plugins/content/emailcloak/src/Extension/EmailCloak.php [hunk 1]
```diff
@@ -62,6 +62,10 @@ public function onContentPrepare(ContentPrepareEvent $event)
             return;
         }
 
+        if (\is_null($item->text)) {
```
Comment (bembelimen) [inline plugins/content/emailcloak/src/Extension/EmailCloak.php:65]:
```suggestion
        if (!isset($item->text)) {
```

Comment (bembelimen) [reply plugins/content/emailcloak/src/Extension/EmailCloak.php:65]: Probably using ```isset``` to avoid notice when key not set?

**Comment by bembelimen** [comment]
Thx

**Pull request merged**